src/main/java/uk/gov/pay/products/validations/ProductsMetadataRequestValidator.java

```diff
@@ -25,15 +25,16 @@
 
     public Optional<Errors> validateCreateRequest(JsonNode payload, List<ProductMetadata> existingMetadataList) {
         if (existingMetadataList.size() >= MAX_NUMBER_OF_METADATA_ALLOWED) {
-            return Optional.of(Errors.from(MAX_NUMBER_OF_METADATA_ALLOWED_ERROR_MSG));
+            return Optional.of(Errors.from(MAX_NUMBER_OF_METADATA_ALLOWED_ERROR_MSG, "MAX_METADATA_LENGTH_EXCEEDED"));
```

Comment:
If this pattern is used in other places across products it might be good to extract this to an enum or something.